Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -21,14 +21,32 @@ public interface IClearCrashClick
         void ClearCrashButton();
     }
 
+    enum PlatformType
+    {
+        Uwp,
```

Comment:
Xamarin.Forms has that already, please use https://docs.microsoft.com/en-us/xamarin/xamarin-forms/platform/device